task2.py

```diff
@@ -0,0 +1,44 @@
+names = ['Smith', 'Johnson', 'Williams', 'Jones', 'Brown', 'Davis', 'Miller', 'Wilson', 'Moore', 'Taylor', 'Anderson', 'Thomas', 'Jackson', 'White']
+
+# Print the entire list
+print("Names:",names)
+
+# Print the length of each element of the list
+Name_length = [len(name) for name in names]
+print("Name lengths : ",Name_length)
+
+sorted_len = list(set(Name_length))
```

Comment:
Change variable name to unique_len_list because it is not sorted list it is list with unique lengths. After changing it, you can merge the PR.